frontend/src/views/Profile.vue

```diff
@@ -3,12 +3,28 @@
     <content-card max-width="800" :title="$tc('views.profile.profile') + ': ' + user.displayName" toolbar>
       <v-col>
         <v-skeleton-loader type="text" :loading="profile._meta.loading">
-          <api-text-field
-            :name="$tc('entity.user.fields.email')"
-            :uri="profile._meta.self"
-            fieldname="email"
-            :editing="false"
-            required />
+          <v-row class="e-form-container">
+            <v-col cols="10">
+              <api-text-field
+                :name="$tc('entity.user.fields.email')"
+                :uri="profile._meta.self"
+                fieldname="email"
+                :editing="false"
```

Comment:
```suggestion
                disabled
```

`editing` property existiert nicht auf `ApiTextField`

Reply:
ich hab 'readonly' gewählt. 
bei 'disabled' wird die Farbe vom Control anders - was seltsam ausseiht.